test/hook/context/generator.go

```diff
@@ -33,17 +30,19 @@ type BindingContextController struct {
 	ScheduleManager   schedulemanager.ScheduleManager
 
 	UpdateTimeout time.Duration
+
+	fakeCluster *fake.FakeCluster
 }
 
 func NewBindingContextController(config string) (*BindingContextController, error) {
-	FakeCluster = fake.NewFakeCluster()
-
 	return &BindingContextController{
 		HookMap:    make(map[string]string),
 		HookConfig: config,
 
 		Controller:    NewStateController(),
 		UpdateTimeout: 1500 * time.Millisecond,
+
+		fakeCluster: fake.NewFakeCluster(fake.ClusterVersionV119),
```

Comment:
It seems that BindingContextController always uses 1.19 and there is no way to change this version in tests.

Reply:
Fixed